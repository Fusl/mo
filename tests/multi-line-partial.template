Partial:

{{> multi-line-partial.partial}}

Indented:

    {{> multi-line-partial.partial}}
