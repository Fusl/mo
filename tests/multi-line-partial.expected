Partial:

line 1
line 2

Indented:

    line 1
    line 2
